packages/camera/camera/example/integration_test/camera_test.dart

```diff
@@ -102,14 +100,12 @@ void main() {
         'Capturing video at $preset (${expectedSize.width}x${expectedSize.height}) using camera ${controller.description.name}');
 
     // Take Video
-    final String filePath =
-        '${testDir.path}/${DateTime.now().millisecondsSinceEpoch}.mp4';
-    await controller.startVideoRecording(filePath);
+    await controller.startVideoRecording();
     sleep(const Duration(milliseconds: 300));
-    await controller.stopVideoRecording();
+    final file = await controller.stopVideoRecording();
```

Comment:
The previous implementation also used two separate futures for the `startVideoRecording` and the `stopVideoRecording`. Meaning `startVideoRecording` completed immediately once the native side started recording. Users then had to make a separate call to `stopVideoRecoding` and await it before they could access the file. 

I think the current implementation is easier to use since you don't have to prepare a path in advance. The only disadvantage is that you need to move the file as a developer yourself if you don't want it saved on the location we choose (this of course only goes for the Android and iOS implementations).

Reply:
Yes, the current version is better for web as well, thanks clarifying the mobile perspective, I was concerned about that mostly!